Generator/Generator/Arguments.swift

```diff
@@ -258,3 +262,49 @@ func generateArgPrepare (isVararg: Bool, _ args: [JGodotArgument], methodHasRetu
     }
     return (body, withUnsafeCallNestLevel)
 }
+
+#else
+
+func generateArgPrepare (isVararg: Bool, _ args: [JGodotArgument], methodHasReturn: Bool) -> (String, Int) {
+    var body = ""
+    var withUnsafeCallNestLevel = 0
+    let retFromWith = methodHasReturn ? "return " : ""
+    
+    // TODO: this case should get the same treatment as a second branch.
+    if isVararg {
```

Comment:
Could you explain what you had in mind for this?

The original code has this condition:

```
isVararg || args.count > 0
```

For this block, and here it seems to be handled differently, so I am curious for the rationale here.

There is additionally some check duplication here, which was necessary on the original code, but now looks redundant (the nested `if isVararg`):

Reply:
Yes, it's indeed a redundant check now. That's an oversight, will fix.

What I had in mind is that I still haven't figured out the difference between `isVararg` being true and false. So when it's true, the original implementation is used.

For a case when it's false and there are arguments, I use the new approach with on-stack storage.

Reply:
Fixed